### PR TITLE
Spectral blade now shows orbs, not ghosts

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Tweaks the spectral blade to show small orbs to represent the ghosts orbiting it, rather than the ghosts themselves. This means no concerns about ghosts being stuck visible, or ghosts pointing. The only thing the living will see are the little orbs.
The orbs will still have the colors of the ghosts, meaning they will still have a bit of character, but they will not have any of the same levels of identifiability as the original characters.

## Why It's Good For The Game
The spectral blade is one of the few places in the game where the dead can be manifested by the living. Ghost invisibility has been kind of a problem with ghosts getting stuck visible due to weird interactions with the spectral blade, which leads to ghosts travelling everywhere across the station. Pointing also leads to some pretty troublesome interactions of people pointing out antags or whatnot.

This solves all of those problems, and does it in kind of a cute, charming way, I think.

## Images of changes
image

## Testing
Loaded in, swung the sword around a little, orbited it. Would like to see this TMed for at least a round just to load-test it.

## Changelog
:cl:
tweak: Ghosts are now no longer visible when orbiting the spectral blade, instead manifesting in the form of orbs.
/:cl:

### CODE REVIEW

**Comment by BAGELMENSK** [comment]
but how will I tell grey miners to take a shower

File: code/modules/mining/lavaland/loot/tendril_loot.dm [hunk 1]
```diff
@@ -246,6 +246,7 @@
 	light_range = 7
 	layer = ABOVE_ALL_MOB_LAYER
 
+
```
Comment (Henri215) [inline code/modules/mining/lavaland/loot/tendril_loot.dm:249]:
```suggestion
```

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

**Comment by spacegeese** [comment]
pointing is the funniest part about it though, maybe make it tied to the boo cooldown?

**Comment by lewcc** [comment]
> pointing is the funniest part about it though, maybe make it tied to the boo cooldown?

that's also one of the most problematic things about it (ex. people pointing to antagonists).

**Review by S34NW — APPROVED** [review]

**Pull request merged**